drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c

```diff
@@ -149,6 +149,13 @@ enum nrf_wifi_status umac_cmd_init(struct nrf_wifi_fmac_dev_ctx *fmac_dev_ctx,
 	umac_cmd_data->tcp_ip_checksum_offload = 1;
 #endif /* CONFIG_NRF700X_TCP_IP_CHECKSUM_OFFLOAD */
 	umac_cmd_data->discon_timeout = CONFIG_NRF_WIFI_AP_DEAD_DETECT_TIMEOUT;
+#ifdef CONFIG_NRF_WIFI_RPU_RECOVERY
+	umac_cmd_data->watchdog_timer_val =
+		(CONFIG_NRF_WIFI_RPU_RECOVERY_PS_ACTIVE_TIMEOUT_MS) / 1000;
+#else
+	/* Disable watchdog */
+	umac_cmd_data->watchdog_timer_val = 0xFFFFFF;
```

Comment:
Is this expected to be just 6 `F's` or a typo?

Reply:
Expected, it is a 24-bit counter.